bitnami/apache/values.yaml

```diff
@@ -503,25 +503,11 @@ ingress:
   ##   cert-manager.io/cluster-issuer: cluster-issuer-name
   ##
   annotations: {}
-  ## @param ingress.tls [array] Enable TLS configuration for the hosts defined
+  ## @param ingress.tls Enable TLS configuration for ingress.hostname
   ## ref: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
   ##
-  tls:
-    - hosts:
-        - example.local
-      secretName: example.local-tls
-  ## DEPRECATED: Use ingress.annotations instead of ingress.certManager
-  ## certManager: false
-  ##
+  tls: false
```

Comment:
This change is introducing a breaking change for someone that for instance installs the chart with the following values, right?

```yaml
ingress:
  enabled: true
  selfSigned: true
```

```bash
$ helm template ./bitnami/apache --set ingress.enabled=true,ingress.selfSigned=true
...
  tls:
    - hosts:
        - "example.local"
      secretName: example.local-tls
```

With your changes, the `tls` part will not be rendered unless you set `ingress.tls=true`, is that right?

Maybe a note should be added in the README: https://github.com/bitnami/charts/tree/main/bitnami/apache#upgrading

Reply:
You are right! I'll look into it. 👍

Reply:
I've pushed a change now that I believe fixes this issue. The behavior when using `--set 'ingress.enabled=true,ingress.selfSigned=true'` should now be unchanged, if I didn't mess something up. Please help me double check the logic again. :)

Reply:
I was trying to keep backwards compatibility but then I thought it would be better to do a full standardization of the ingress, as we are already bumping the major version.